.gitignore

```diff
@@ -39,3 +39,8 @@ out/
 ### VS Code ###
 .vscode/
 .env
+
+
+# Exclude creds.yaml
+creds.yaml
+.DS_Store
```

Comment:
.DS_Store is already in line 8

Where is the creds.yaml coming from?

Reply:
Hello Sandra, I used to include API key into app properties from separate file on another project but it didn't work this time :(